equinix/fabric_connection_read_schema.go

```diff
@@ -0,0 +1,708 @@
+			Elem: &schema.Resource{
+				Schema: readAccessPointInterface(),
+			},
+		},
+		"seller_region": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Access point seller region",
+		},
+		"peering_type": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Peering Type",
+		},
+		"provider_connection_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Provider assigned Connection Id",
+		},
+	}
+}
+
+func readFabricConnectionSideSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"service_token": {
+			Type:        schema.TypeSet,
+			Computed:    true,
+			Description: "For service token based connections, Service tokens authorize users to access protected resources and services. Resource owners can distribute the tokens to trusted partners and vendors, allowing selected third parties to work directly with Equinix network assets",
+			Elem: &schema.Resource{
+				Schema: readServiceTokenSch(),
+			},
+		},
+		"access_point": {
+			Type:        schema.TypeSet,
+			Computed:    true,
+			Description: "Point of access details",
+			Elem: &schema.Resource{
+				Schema: readConnectionSideAccessPointSch(),
+			},
+		},
+	}
+}
+
+func readRedundancySch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"group": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Redundancy group identifier",
+		},
+		"priority": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Priority type",
+		},
+	}
+}
+
+func readPortRedundancySch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"priority": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Priority type",
+		},
+	}
+}
+
+func readChangeLogSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"created_by": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Created by User Key",
+		},
+		"created_by_full_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Created by User Full Name",
+		},
+		"created_by_email": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Created by User Email Address",
+		},
+		"created_date_time": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Created by Date and Time",
+		},
+		"updated_by": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by User Key",
+		},
+		"updated_by_full_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by User Full Name",
+		},
+		"updated_by_email": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by User Email Address",
+		},
+		"updated_date_time": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by Date and Time",
+		},
+		"deleted_by": {
```

Comment:
I'm curious about this. We generally don't expect to find deleted resources in Terraform, but this is not a resource, but a resource log?

Reply:
As discussed, API returns based on uuid, irrespective of connection status.